app/views/users/logix/search.html.erb

```diff
@@ -0,0 +1,39 @@
+<link href="/css/search.css" rel="stylesheet">
+
+<%= render "logix_assets/search_bar" %>
+
+<% if @users.empty? %>
+  <div class="container">
+    <h4 class="results-head">No Results Found</h4>
+  </div>
+<% else %>
+  <div class=""></div>
+  <div class="container">
+    <% @users.map { |user| ProfileDecorator.new(user) }.each do |profile| %>
```

Comment:
make @users as decorated users in controller itself

Reply:
Doing this in the controller creates problems with the paginator 😓